src/platform/base.ts

```diff
@@ -0,0 +1,20 @@
+import { NextWebVitalsMetric } from 'next/app';
+import { RequestReport } from '../logger';
+import { EndpointType } from '../shared';
+
+export default interface PlatformConfigurator {
```

Comment:
Can we add a comment explaining this interface?

Reply:
Shouldn't this be `interface Provider { ... }`?

Reply:
done